app/managers.py

```diff
@@ -0,0 +1,51 @@
+import sqlite3
+import os
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(
+            os.path.join(os.path.dirname(__file__), "identifier.sqlite")
+        )
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"""
+            INSERT INTO {self.table_name} (first_name, last_name)
+            VALUES (?, ?)
+            """,
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> None:
+        actor_manager_cursor = self._connection.execute(
+            f"""SELECT * FROM {self.table_name}"""
+        )
+        return [
+            Actor(*row) for row in actor_manager_cursor
+        ]
+
+    def update(
+        self,
+        id_to_update: int,
+        new_first_name: str,
+        new_last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"""UPDATE {self.table_name} """
+            f"""SET first_name = ?, last_name = ? """
+            f"""WHERE id = ?""",

```

Comment:
No need to put triple quotes in each line, the main idea put it in the beginning and in the end, chek everywhere